R/calculate.R

```diff
@@ -108,7 +111,7 @@ calculate_exposure_by_technology_and_company <- function(asset_type,
 
   exposure_by_technology_and_company <- input_data_list$pacta_results %>%
     dplyr::filter(
-      .data$year == .env$start_year,
+      .data$year == .env$start_year + .env$time_horizon,
```

Comment:
I take it this part was sneaked into the PR based on a discussion on slack? I would like to kindly encourage you to:
- revert affected change in this PR
- open a new PR inlcuding only that change
- running a change check or snapshot test on it to quantify the impact

Background: This particular change is small and easy to review, so I could elt it go, but it will affect all results I guess. This makes it impossible to test expectation on changes of the tmsr smsp fix.

Reply:
done https://github.com/2DegreesInvesting/r2dii.climate.stress.test/pull/316